appveyor.yml

```diff
@@ -43,8 +43,7 @@ build_script:
   - if "%config%"=="libretro Release" msbuild libretro\libretro-win32.vcxproj /t:build /p:Configuration="%config%";Platform="%platform%" /m /logger:"C:\Program Files\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
 
 after_build:
-  - ps: $env:gitrev = git describe --tags
-  - ps: $env:my_version = "$env:gitrev"
```

Comment:
What did `$env:gitrev` show and why is it should it be changed to `version-commit` format?